src/components/ContactForm/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useRef, useState } from "react";
+import emailjs from "emailjs-com";
+import {
+  Container,
+  Form,
+  FormInput,
+  InputError,
+  RodoInformation,
+  Message,
+  SendButton,
+  ReCAPTCHAContainer,
+  NameInputContainer,
+  EmailInputContainer,
+  MessageInputContainer,
+  Label,
+} from "./style";
+import { useForm } from "react-hook-form";
+import ReCAPTCHA from "react-google-recaptcha";
+
+type FormValues = {
+  name: string;
+  email: string;
+  message: string;
+};
+
+const Contact = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const [isNotRobot, setIsNotRobot] = useState(false);
+  const captchaRef = useRef<ReCAPTCHA | null>(null);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    setValue,
+  } = useForm<FormValues>();
+  const onSubmit = handleSubmit((data) => {
+    const { name, email, message } = data;
+    console.log(data);
+    if (isNotRobot && name && email && message) {
+      const serviceId = process.env.REACT_APP_SERVICE_ID as string;
```

Comment:
usually commercial projects (although this isn't the rule), use packages or simple helper to read these variables. You can think about this in the future. In addition, I recommend moving such declarations to the top of the file :)